analyzer/javacg-opal/src/test/java/eu/fasten/analyzer/javacgopal/OPALPluginTest.java

```diff
@@ -35,125 +35,120 @@
 public class OPALPluginTest {
```

Comment:
* Why are all the tests commented out?

Reply:
I initially commented them all out to avoid fixing them while the new architecture was experimental, and completely forgot to fix them at the end. Now both OPAL and WALA plugins tests are fixed